app/models/miq_user_role.rb

```diff
@@ -37,8 +37,8 @@ def feature_identifiers
   def allows?(identifier:)
     if feature_identifiers.include?(identifier)
       true
-    elsif parent = MiqProductFeature.parent_for_feature(identifier)
-      allows?(:identifier => parent.identifier)
+    elsif (parent = MiqProductFeature.feature_parent(identifier))
+      allows?(:identifier => parent)
```

Comment:
So it worked before because we were passed an actual feature, now we're given an identifier.